examples/options/src/options/common_options.cpp

```diff
@@ -26,6 +26,9 @@ traccc::common_options::common_options(po::options_description& desc) {
     desc.add_options()("check-performance",
                        po::value<bool>()->default_value(false),
                        "generate performance result");
+    desc.add_options()("perform-ambiguity-resolution",
+                       po::value<bool>()->default_value(true),
+                       "perform ambiguity resolution");
```

Comment:
Do we need this option? 🤔 We can't turn algorithms on/off generally.

I've been pondering since a longer time about how to introduce a very minimal amount of "framework functionality" into the project, to make it easy to define different algorithm sequences. Without having to write a bunch of separate `.cpp` files.

But for now, we have "frozen algorithms sequences" in all of our executables. So let's just add the ambiguity resolution to the "seeding example", like Beomki recommended. And just run this algorithm all the time. 🤔 I wouldn't be afraid of that...

Reply:
I suggested to have this option temporarily until we implement the GPU version of the ambiguity solver